beaver/src/export.js

```diff
@@ -1,63 +1,63 @@
 /* global FLBuilder */
 
-if ( parseInt( window.tiTpc.tier ) === 3 ) {
-	const contextMenu = document.getElementById( 'tmpl-fl-row-overlay' );
-
-	const tpcExport = ( e ) => {
-		const row = e.closest( '.fl-row' );
-		const node = row.dataset.node;
-
-		const message = `<div class="tpc-template-cloud-export-modal">
-		<h1>${ window.tiTpc.exporter.modalLabel }</h1>
-		<label for="tpc-${ node }">${ window.tiTpc.exporter.textLabel }</label>
-		<input id="tpc-${ node }" type="text" placeholder="${ window.tiTpc.exporter.textPlaceholder }" />
-	</div>`;
-
-		FLBuilder.confirm( {
-			message,
-			ok: () => {
-				const input = document.getElementById( `tpc-${ node }` );
-				const title = input.value || 'Template';
-				setTimeout( function () {
-					FLBuilder.showAjaxLoader();
-					FLBuilder.ajax(
-						{
-							action: 'ti_export_template',
-							node,
-							title,
-						},
-						( res ) => {
-							if ( undefined !== res.success && ! res.success ) {
-								FLBuilder.alert(
-									`<h1>${ window.tiTpc.exporter.exportFailed }</h1> ${ res.data }`
-								);
-							}
-
-							FLBuilder.hideAjaxLoader();
+const contextMenu = document.getElementById( 'tmpl-fl-row-overlay' );
+
+const tpcExport = ( e ) => {
+	const row = e.closest( '.fl-row' );
+	const node = row.dataset.node;
+
+	const message = `<div class="tpc-template-cloud-export-modal">
+	<h1>${ window.tiTpc.exporter.modalLabel }</h1>
+	<label for="tpc-${ node }">${ window.tiTpc.exporter.textLabel }</label>
+	<input id="tpc-${ node }" type="text" placeholder="${ window.tiTpc.exporter.textPlaceholder }" />
+</div>`;
+
+	FLBuilder.confirm( {
+		message,
+		ok: () => {
+			const input = document.getElementById( `tpc-${ node }` );
+			const title = input.value || 'Template';
```

Comment:
Can we also have this default value as a translatable string via `window.tiTpc.exporter`?